code/ruby/index_ranges.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'xapian'
+require 'json'
+require_relative 'support'
+
+def index(data_path, db_path)
+  # puts "#{data_path} #{db_path}"
```

Comment:
I assume this (and similar) are leftovers from debugging so I've pushed a commit to remove them.